packages/main/test/samples/Dialog.sample.html

```diff
@@ -197,4 +197,102 @@ <h3>Dialog with SAP Fiori Styled Footer</h3>
+			</div>
+		</ui5-dialog>
+
+		<ui5-dialog id="information-state-dialog" header-text="Information" state="Information">
+			<p>Information state dialog</p>
+			<div slot="footer" style="display: flex; justify-content: flex-end; width: 100%; padding: .25rem 1rem;">
+				<ui5-button id="information-close">Close</ui5-button>
+			</div>
+		</ui5-dialog>
+
+		<ui5-dialog id="success-state-dialog" header-text="Success" state="Success">
+			<p>Success state dialog</p>
+			<div slot="footer" style="display: flex; justify-content: flex-end; width: 100%; padding: .25rem 1rem;">
+				<ui5-button id="success-close">Close</ui5-button>
+			</div>
+		</ui5-dialog>
+
+		<ui5-dialog id="warning-state-dialog" header-text="Warning" state="Warning">
+			<p>Warning state dialog</p>
+			<div slot="footer" style="display: flex; justify-content: flex-end; width: 100%; padding: .25rem 1rem;">
+				<ui5-button id="warning-close">Close</ui5-button>
+			</div>
+		</ui5-dialog>
+	</div>
+
+	<script>
+		var dialogOpenerError = document.getElementById("error-state");
+		var dialogError = document.getElementById("error-state-dialog");
+		var dialogCloserError = document.getElementById("error-close");
+
+		dialogOpenerError.addEventListener("click", function () {
+			dialogError.show();
+		});
+
+		dialogCloserError.addEventListener("click", function () {
+			dialogError.close();
+		});
+
+		var dialogOpenerInfo = document.getElementById("information-state");
+		var dialogInfo = document.getElementById("information-state-dialog");
+		var dialogCloserInfo = document.getElementById("information-close");
+
+		dialogOpenerInfo.addEventListener("click", function () {
+			dialogInfo.show();
+		});
+
+		dialogCloserInfo.addEventListener("click", function () {
+			dialogInfo.close();
+		});
+
+		var dialogOpenerSuccess = document.getElementById("success-state");
+		var dialogSuccess = document.getElementById("success-state-dialog");
+		var dialogCloserSuccess = document.getElementById("success-close");
+
+		dialogOpenerSuccess.addEventListener("click", function () {
+			dialogSuccess.show();
+		});
+
+		dialogCloserSuccess.addEventListener("click", function () {
+			dialogSuccess.close();
+		});
+
+		var dialogOpenerWarning = document.getElementById("warning-state");
+		var dialogWarning = document.getElementById("warning-state-dialog");
+		var dialogCloserWarning = document.getElementById("warning-close");
+
+		dialogOpenerWarning.addEventListener("click", function () {
+			dialogWarning.show();
+		});
+
+		dialogCloserWarning.addEventListener("click", function () {
+			dialogWarning.close();
+		});
+
+	</script>
+
+	<pre class="highlight">{% highlight html %}
+		<ui5-dialog id="error-state-dialog" header-text="Error" state="Error">
+			<p>Error state dialog</p>
+			<div slot="footer" style="display: flex; justify-content: flex-end; width: 100%; padding: .25rem 1rem;">
+				<ui5-button id="error-close">Close</ui5-button>
+			</div>
+		</ui5-dialog>
```

Comment:
Remove 2 tabs of indentation of every line. Do this for the above sample about Fiori footer too.